README.md

```diff
@@ -1,99 +1,355 @@
 Run `make verison-patch` to update the version after implementing fixes for bugs etc - this will not update the api path. This is used to track fixes for testing etc. 
 
-#### Running The Project Locally (Docker)
 
-Use the make commands under the `docker` section to start the project.
+## Running Tests
 
-Install dependencies - `yarn`
-Run Project - `make build`
+Build the test container:
+```bash
+make run-test
+```
 
-#### Gettings Access to tools and credentials
+Run tests:
+```bash
+make run-test-pipeline
+```
 
-to run the project for development, the following is bare requirement:
+## Deployment
+Build backend:
+```bash
+make build-backend
+```
 
-- aws configuration for aws-pcc and minio [Example](./.config/credentials.example)
-- rclone configuration [Example](./.config/rclone.conf.example)
-- env file configuration [Example](./.config/.env.example)
+Terraform:
+```bash
+make plan
+```
 
-Refer [here](./docs/access.md)
+```bash
+make apply
+```
+Tag Based Deployments:
+```bash
+make tag-<env>
+```
 
-#### Populate the database
+## Documentation
 
-- Ensure you are connected to the cisco VPN
-- Make sure the project is up and running by using the docker make commands
-- Run `make migration-run` for the database to be setup.
-- Run `make sync` to update your local environment with all the necessary files
-- Run `make parse` to populate the db
+#### Developer Documentation
+The developer docs (compodoc) can be generated by running the following command:
 
-## Developer Docs
+```bash
+make build
+```
+```bash
+make dev-docs
+```
+The docs are a work in progress. Configuration options can be found [here](https://compodoc.app/guides/options.html).
 
-- Once the project has been configured and all dependencies installed, after running `make build` (ensure project is running with docker ps) you can run `make dev-docs` to view compodoc/jsdocs generated documentation for the apps/backend application
-- This takes a few minutes to build.
-- Visit `http://localhost:8081` in the brower to view the docs
-- To suppress the logs or make other configuration changes edit the compodoc.yaml file in the apps/backend directory.
-- configuration options can be found [here](https://compodoc.app/guides/options.html)
+#### Project Documentation
 
-### Reconciliation
+The project docs can be generated by running the following commands:
 
-- Make sure the project is up and running by using the docker make commands
-- Make sure you have completed the steps above
-- Edit the `apps/backend/fixtures/reconcile.json` file to specifiy config
-- Run (from root) `make reconcile`
+```bash
+make build
+```
+```bash
+make build-docs
+```
 
-### Reporting
 
-- Make sure the project is up and running by using the docker make commands
-- Make sure you have completed the steps above
-- Edit the `apps/backend/fixtures/report.json` file to specifiy config
-- Run (from root) `make report`
+#### Database Documentation
+- [Database Documentation](https://dbdocs.io/chelsea-EYDS/bcpcc)
```

Comment:
wonder if we should log a ticket to find these a permanent home?

Reply:
I can make a ticket to try and bring up at the next refinement?